docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :bug:`9730` Fix the issue where the app breaks when the user clicks on "Ethereum Staking" in the blockchain balance summary on the dashboard.
+* :bug:`-` Fixed the issue where addresses existing only on L2 networks were not being checked for airdrop eligibility.
```

Comment:
Changelog should be user facing. Please fix this and the other such mistakes in the changelog. I think the frontend makes them.

```suggestion
* :bug:`-` Addresses tracked only on non-mainnet EVM networks should now also be checked for airdrop eligibility.
```